tests/models/test_anthropic.py

```diff
@@ -589,6 +589,21 @@ async def test_image_url_input_invalid_mime_type(allow_model_requests: None, ant
     )
 
 
+@pytest.mark.vcr()
+async def test_image_as_binary_content_tool_response(
+    allow_model_requests: None, anthropic_api_key: str, image_content: BinaryContent
+):
+    m = AnthropicModel('claude-3-5-sonnet-latest', provider=AnthropicProvider(api_key=anthropic_api_key))
+    agent = Agent(m)
+
+    @agent.tool_plain
+    async def get_image() -> BinaryContent:
+        return image_content
+
+    result = await agent.run(['What fruit is in the image you can get from the get_image tool?'])
+    assert 'kiwi' in result.output
```

Comment:
Can we use `snapshot()`? I can explain how to use it if you haven't used it before.